server/antibodyapi/default_config.py

```diff
@@ -30,6 +30,8 @@ class DefaultConfig:
 
     ANTIBODY_ELASTICSEARCH_INDEX = 'hm_antibodies'
     QUERY_ELASTICSEARCH_DIRECTLY = False
+    CEDAR_VALIDATION_URL = "https://api.metadatavalidator.metadatacenter.org/service/validate-tsv"
```

Comment:
Move this new config to `example.app.conf` so we can configure it in `app.cpnf` on the deployment environments. This `default_config.py` is mainly used for testing with default values, a few new values added to `app.conf` don't exist in the default config file.